manifests/init.pp

```diff
@@ -14,6 +14,9 @@
 # [*hostname*]
 #   String. Override default hostname used to identify this agent.
 #
+# [*omit_hostname*]
+#   Boolean. Do no set the "host" tag in the telegraf agent.
```

Comment:
Nitpick:  Minor typo - should be "not" instead of "no" ;)  It's there in the source code for Telegraf as well, I see.